### PR TITLE
add(tools): create-tailwind-plugin

### PR DESCRIPTION
**Pull request by Landish**
| Name                 | Link                 |
| -------------------- | -------------------- |
| create-tailwind-plugin | https://github.com/Landish/create-tailwind-plugin |

create-tailwind-plugin - Create Tailwind CSS plugin and publish on npm with ease

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Landish** [comment]
Wasn't quite sure the category should be Tools or Starters & Themes (cli/generator). Currently its under Tools.

File: README.md [hunk 1]
```diff
@@ -202,6 +202,7 @@
 - 🌍 [Tailwind Cheat Sheet](http://nerdcave.com/tailwind-cheat-sheet) - Tailwind CSS class names cheat sheet.
 - 🌍 [Tailwind Cheat Sheet](https://github.com/LeCoupa/awesome-cheatsheets/blob/master/frontend/tailwind.css) - Tailwind CSS class names in one single file.
 - 🌍 [Tailwind Cheat Sheet](https://umeshmk.github.io/Tailwindcss-cheatsheet) - Tailwind CSS class names, variants and directives cheat sheet.
+- 🔧 [create-tailwind-plugin](https://github.com/Landish/create-tailwind-plugin) - Create Tailwind CSS plugin and publish on npm with ease.
```
Comment (innocenzi) [inline README.md:205]: Can you put group it under the other 🔧s please?

**Review by innocenzi — CHANGES_REQUESTED** [review]
It's in the right category, but it's not grouped with the other 🔧s. 

Looks like quite a useful project though. You should add a TypeScript option. 👍

File: README.md (hunk 1 above)
Comment (Landish) [reply README.md]: is this one correct place? 
under Protoship Codegen?

File: README.md [hunk 2]
```diff
@@ -198,11 +198,11 @@
 - 💼🔧 [Laravel Form Components](https://github.com/pascalbaljetmedia/laravel-form-components) - Blade form components using Tailwind CSS Custom Forms.
 - 🔧 [re-tailwind](https://github.com/phthhieu/re-tailwind) - ReasonML utility that generates Tailwind classes.
 - 🔧 [Protoship Codegen](https://protoship.io) - Code generator that creates Tailwind CSS based HTML & CSS from Sketch designs.
+- 🔧 [create-tailwind-plugin](https://github.com/Landish/create-tailwind-plugin) - Create Tailwind CSS plugin and publish on npm with ease.
```
Comment (innocenzi) [inline README.md:201]:
```suggestion
- 🔧 [create-tailwind-plugin](https://github.com/Landish/create-tailwind-plugin) - Plugin scaffolder for Tailwind CSS.
```

**Review by innocenzi — APPROVED** [review]

**Pull request merged**